lib/waffle/definition/storage.ex

```diff
@@ -116,6 +136,7 @@ defmodule Waffle.Definition.Storage do
       def acl(_, _), do: @acl
       def s3_object_headers(_, _), do: []
       def async, do: @async
+      def remote_file_headers(_), do: []
```

Comment:
we have a warning on that line

```
warning: this clause cannot match because a previous clause at line 27 always matches
  test/actions/store_test.exs:21
```